101-azd-useBearerToken/run.ps1

```diff
@@ -0,0 +1,6 @@
+# you can copy and paste this code into your Azure Pipeline build
+# make sure and enable OAuth token access for the Classic Editor
+Install-Module VSTeam -Scope CurrentUser -Force
+Set-VSTeamAccount -Account $(System.CollectionUri) -Token $(System.AccessToken) -UseBearerToken
```

Comment:
In this case, it works nicely without parameter, but for easier use, I think we should encourage to always use script parameters and not define hard-coded variables in the script.

Reply:
Passing arguments to a script is only an option if you select "File Path". If you select "Inline" the arguments are not an option.  If they just copy and paste the text to and Inline PowerShell task the parameters will not work. Maybe we need to offer both options. One for those that want to use the file and those that want to just copy and paste. Thoughts?

Reply:
You could make the values that you use as default in the parameters and the parameters are always not mandatory.
In that way you can just run the script or use the parameters.